### DISCUSSION TITLE
Open Shell for Windows 11

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#894] [General]
Please I would like to know when it's going to release version for Windows 11?

**Comment by NK2U** [comment]
Right now.  I have had it installed since Win 11 first went live on the Dev channel.

**Reply by ghost** [reply]
How do I access the Dev channel? I just created a Github account thinking I would gain access, but I'm lost.

**Reply by dntfkncarebear** [reply]
heya, just signed up...not sure what to do here but my windows updated and now I am back to this horrid app-like situation.... help?

**Reply by dntfkncarebear** [reply]
current download won't install, not sure if it's an issue on my end or if i need to grab a new download that i can't find :(

**Comment by Dustyn69** [comment]
In addition, I am curious about the future of OpenShell.
Is anyone actively working on it?
Or is it going the same route as ClassicShell?

When and if it comes to updating to Windows 11, it would be nice if OpenShell was properly updated for full Windows 11 support.

**Comment by ofigustavo** [comment]
Hello. I am using the last compilation of windows 11: And today, 2 december 2021 is the Windows 11 Insider Preview Build 22509.Openshell works very well.

**Reply by MichaelIsGreat123** [reply]
Thanks for the feedback, ofigustavo. I hope it will be the same also when you use a normal build of Windows 11.

**Comment by blackcrack** [comment]
@ge0rdi should discuss here too..

**Comment by MEdson71** [comment]
Hello.
I just found this article and peple may find it useful.

https://www.askvg.com/tip-restore-old-classic-start-menu-in-windows-11/

**Comment by tallman69** [comment]
I have been running Open Shell v4.4.169-beta on the latest build of Windows 11 with no problems.  However, before I installed Open Shell I installed ExporerPatcher and configured it to my liking.  ExplorerPatcher automatically performs the tasks, plus many more, outlined in the ASKVG link posted by MEdson71 above.  The two programs seem to work well together.  You can find ExplorerPatcher here:

https://github.com/valinet/ExplorerPatcher

**Reply by ofigustavo** [reply]
Windows 11 Insider Preview 22572.1 (ni_release). dont work. Be careful with this program:EXPLORERPATCHER.  I had to reset my computer for 2 times. I had to unistall this software. Screen all in black. my computer was blocked. I recommend not install this software.

**Reply by tallman69** [reply]
I am running ExplorerPatcher on the latest stable build of Windows 11 without any problems.  The developer of ExplorerPatcher says that he does not test the software on Insider Preview releases.

https://github.com/valinet/ExplorerPatcher/discussions/1055

**Comment by YesTrustMe** [comment]
I'm going to bump this thread. I know many are looking for a stable non beta (or nightly) build for W11.

**Comment by blackcrack** [comment]
if you know anyone where be in able to programming here, speak with him.
i guess well programmed patches be welcome.
Speak with the Peoples in the real live if they Programming
by the way, if anyone comes here and add a Donation-Icon and a couples Screenshot, could this become something..
=> ![Download-Counter](https://img.shields.io/github/downloads/Open-Shell/Open-Shell-Menu/total?style=for-the-badge)

1.8 Millions downloads !
and why do not add a Donation Button ? @Ibuprophen @ge0rdi i bet it would rational and meaningful 

best

**Comment by NK2U** [comment]
That would be most welcome!  Thanks!
    
      
      
      
      
      
      
      
      
		@page { margin: 0.79in }
		p { margin-bottom: 0.1in; line-height: 115% }
		a:link { so-language: zxx }
	
      Roland Stiner,
        NK2U
      ***@***.***
      
      
      
      
      
      
    
    On 4/26/2022 7:41 PM, YesTrustMe wrote:
    
    
      
      I'm going to bump this thread. I know many are
        looking for a stable non beta (or nightly) build for W11.
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you commented.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2642546",
"url": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2642546",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Rick75230** [comment]
I have one copy of Win 11 Home running. It was a fresh install. The installer was created with Rufus for "Doesn't have TMP 2.0". I set it to install it to an MBR, not a GPT. It's an Asus desktop motherboard with a BIOS date 2012.

I have been running Open Shell v4.4.169-beta on the latest build of Windows 11. (I don't normally use that PC, so it's just for testing Win 11.)

Problems I am running into are that the only way to open Open Shell is the Windows key. Although I have it set to open with clicks on the Start Button, etc., none of those work. Also, right-clicking on the Start Button does not pull up the Open Shell settings and other links.

In the Open Shell Settings I did set an alternate key, alt-F12, and that does pull up the menu.

One reason I did that is I normally use Chrome Remote Desktop (CRD) to go into the various PC's from Linux Mint 20.4 with a Mate interface (pronounced "mah-TAY"); Mint is based on Ubuntu.

In Mint, if I press the Windows key it pulls up the Mint equivalent of Microsoft's Start menu even if I am in a CRD session. Pressing my alternate Open Shell activation key opens the Open Shell menu on the remote PC.

--------
**Something I'm looking into is creating a Powershell script** that will press either the Windows key or my alt-F12 alternate, then running that script from a taskbar icon. However I have never worked with Powershell.

**Comment by tallman69** [comment]
Have you tried installing ExplorerPatcher to see if that makes any difference?
https://github.com/valinet/ExplorerPatcher

**Reply by Rick75230** [reply]
No. Basically, I simply did a fresh install of Win 11 to see if with Rufus I could even get Win 11 to run on my PC hardware.

I have worked with Win 11 only a **very** little bit. So far I don't see any features that would particularly interest me. I mainly use Google Chrome and a file manager under Linux on an Asus Chromebox hardware hacked to run Linux. Most of what I do is posting on Facebook or Quora.

Also, on my "real" PC I have a **very** unusual setup with **loads** of junctions to move a bunch of **program** files to a different drive. My different PC's are set to boot to different drives which I can select from UEFI during boot. So I don't want to screw with all that if it will simply be going from Win 10 to Win 11.

Plus, I assume that as additional updates are done on Open-Shell that the problem will be resolved. So I don't want to resort to temporary workarounds like "patcher" programs just to get a beta program running on a test machine.

I basically posted so there would be a "record" of the problem.

Also, "pressing the Windows key while in a Linux Chrome Remote Desktop session" isn't a bug, so I figured I'll post my workaround for that.

**Reply by Rick75230** [reply]
As noted above, I set the following:

1) Open the Classic Shell menu with Windows Key
2) Settings
3) Open-Shell Menu
4) Tab: Controls
5) Additional Classic Menu Hotkey: Alt+F12
6) Ok

I just tried this from a Powershell command line and it works:

[System.Windows.Forms.Sendkeys]::SendWait('%{F12}')

**Comment by NK2U** [comment]
Just downloaded the 170 version; works very well.  No glitches or
    bugs to be found yet.  Thanks again!
    
      
        
        
        
        
        
        
        
        @page { margin: 0.79in }p { margin-bottom: 0.1in; line-height: 115% }a:link { so-language: zxx }
        Roland Stiner,
          NK2U
        ***@***.***
        
        
        
        
        
        
      
      On 4/26/2022 7:41 PM, YesTrustMe
        wrote:
      
      
        
        I'm going to bump this thread. I know many are
          looking for a stable non beta (or nightly) build for W11.
        —
          Reply to this email directly, view it on GitHub, or unsubscribe.
          You are receiving this because you commented.Message
            ID: ***@***.***>
        [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2642546",
"url": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2642546",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by YesTrustMe** [reply]
> Just downloaded the 170 version; works very well.  No glitches or bugs to be found yet.} } ]

How can you say it works well if you just downloaded it and haven't used it for a month or so before posting that it works well? Giveit time. If there was an official build for W11, it would be posted and promoted.

**Comment by MichaelIsGreat123** [comment]
Personally, I plan to install Windows 11 only around the end of September 2022, as Windows 11 is still a work in progress. I regularly read articles about the latest updates for Windows 11 that corrected "a ton of fixes"!! 

I indicate the end of September 2022 deadline only because apparently the deadline for downloading for free Windows 11 will be October 5, 2022 (See the article "Is Microsoft's Free Windows 11 Upgrade Offer Coming to an End? -- Redmond Channel Partner" at https://rcpmag.com/articles/2022/01/26/is-microsoft-free-windows-11-upgrade-offer-coming-to-an-end.aspx ).

But this deadline is not even a deadline at all I believe. There was a deadline to install for free Windows 10 from the previous version of Windows and you can still install it right now in May 2022 for free so Microsoft also gave a deadline for the free installation of Windows 10 but they did not apply this deadline. I believe it will be exactly the same for Windows 11 for those who have Windows 10 and compatible hardware to install Windows 11, as Microsoft really wants the majority of Windows 10 users to move to Windows 11 as soon as possible and it is on top of that a free upgrade.

Now, concerning Open-Shell that I use daily on Windows 10, there are more than 1 million of users of this addon. I am confident that there will be a Windows 11 version of Open-Shell in the future but why would the developer hurries considering the fact that Windows 11 has been a work in progress for now nearly a year?!!

To conclude, be confident that you will see a working version of Open-Shell running without problems on Windows 11 in the future and probably before this deadline of October 5, 2022.

**Reply by blackcrack** [reply]
Open-Shell is not idepended to a Orderingsystem, MS is independet to the Users.. aand Open-Shell iss also independed of the Users, but only if the Users and programmers whant have a Win11 Version must get up the ass ad starting to help here at Openshell to programming with, because there be at moment only 1 Programmer where directs the project imho, we need Peoples who helping with there to programming Openshell to be usable in Win11.. so please, look's for frends who want helping programming here :) 
And not only speak wisely ;)

Thank you
Best

**Comment by MichaelIsGreat123** [comment]
Check this video: "(May 28, 2022) Change the Windows 11 start menu with Open-Shell - May 28, 2022 [by Sam Hayden] - YouTube" at https://www.youtube.com/watch?v=02RWTwZc9K0
In this video, the guy uses Open-Shell 4.4.160 and he has no problems at all installing successfully Open-Shell on Windows 11.

I currently cannot at all test this scenario because I am still on Windows 10. However, if among you, there are people who recently successfully installed Open-Shell on Windows 11, it would be great if you would not mind leaving a comment on this posting and indicate:
---The date you installed Open-Shell on Windows 11?
---Which version of Open-Shell that you have used?
---Have you had any update yet of Windows 11 and has any update of Windows 11 caused any problem with the installed Open-Shell on your Windows 11 computer?
Thanks in advance for any feedback that you will share.

Kind Regards
Michael

**Reply by Rick75230** [reply]
I have a test version of Win 11. I mainly run Win 10.

If you are running a desktop, I suggest what I call "Upgrade ... **AND DON'T**."

To do that:

1) Buy an equivalent SSD/HDD. Most are fairly inexpensive right now.

2) **Note!** If you are using OEM backup software (e.g. Acronis) you need to get the same brand of disk. Otherwise your backup software won't work (as detailed below).

Exception: You can leave the old disk in, so **"a"** disk of the same brand will still be there.

3) **Clone** your OS disk.

4) **Swap** disks.

5) Make sure the clone and the original both fully work.

6) Remove the one you are **not** going to upgrade.

7) Upgrade **one** of them.

That way, if something screws up, you don't have to screw with things like trying to **revert**--you just swap disks.

You can still use the other one for backups, an extra data drive, or whatever.  It's still a lot less of a pain than spending days trying to revert and fix.

---------------------------------
My test Win 11 is a **fresh install** on a **non-compliant** machine. You can install that way with Rufus.

[Rufus](https://rufus.ie/en/)

---------------------------------
I use Open-Shell 4.4.170 and ExplorerPatcher.

I have run into problems with programs completely unrelated to those, so I'll wait for MS to get their act together on Win 11 before using it on any of my "regular" machines. For instance, QTTabBar doesn't work.

For some stupid reason Win 11's file select dialogues and File Manager often don't show files I **know** are there--because a **different** program **is** showing them.

I mainly run Linux Mint on an Asus Chromebox hardware hacked to run Linux Mint and mainly do text postings to Quora (like Reddit), so I don't need to be spending huge amounts of time screwing with Win 11 when all my other cross-connected custom stuff gets things done.

**Comment by Lieuallen** [comment]
Just an FYI: I'm successfully running Open-Shell 4.4.169 and ExplorerPatcher 22000.376.40.21 on Windows 11 21H2 build 22000.739 and have no problems at all. These two extensions made the migration to Windows 11 seamless, and I'm loving it! I've been using Windows 7 since 2010 and was reluctant to leave it. But this switch was easy!

**Comment by MichaelIsGreat123** [comment]
Hello  Lieuallen, thanks a lot for your feedback on this matter. 

That is really great to know because honestly, this Windows 11 start menu, it might be good for beginners who have not used a computer but it is the worst for people who are productive users of Windows.

By the way, just update to the latest version 4.4.170 of Open-Shell and you should not have any problems at all I believe.

Kind Regards
Michael

**Comment by MichaelIsGreat123** [comment]
Hello Lieuallen, I am still curious of what you should do with ExplorerPatcher in order to ensure that your installation of Open-Shell will work without any problems. I was wondering that, if you only install ExplorerPatcher without using any of its functions, would it still allow the installation of Open-Shell to run without any problems at all?

As for the video I mentioned before, I believe the guy did not install ExplorerPatcher at all. Check this video: "(May 28, 2022) Change the Windows 11 start menu with Open-Shell - May 28, 2022 [by Sam Hayden] - YouTube" at https://www.youtube.com/watch?v=02RWTwZc9K0 

Kind Regards
Michael

**Comment by Rick75230** [comment]
As I have indicated above:

1. I used Rufus to do a fresh Win 11 install on non-compliant hardware.
2. I have installed Open-Shell ver 4.4.170.
3. I have installed ExplorerPatcher.

**I found out it is possible to use the Win10 version of File Explorer by using a registry tweak.**
 
1) In Reg edit go to:
	Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell Extensions

2) If that does not contain a registry key (folder) named Blocked:

	a) Right click on \Shell Extensions
	b) create a new Key named: Blocked

3) Click on the Blocked registry key folder

4) Copy this: {e2bf9676-5f8f-435c-97eb-11607a5bedf7}

5) In Blocked right panel right-click > New > String Value

6) Paste the key name to create the key.

7) Double-click the key.

	a) TO USE Win 11 explorer type: < delete > **with** the arrows and **without** the spaces.

	b) TO USE Win 10 explorer type: < added > **with** the arrows and **without** the spaces.

QTTabBar **does** work with the Win10 version.

**Comment by NK2U** [comment]
Any idea when Open Shell will be
      updated?
    
      
      
      
      
      
      
      
      
		@page { margin: 0.79in }
		p { margin-bottom: 0.1in; line-height: 115% }
		a:link { so-language: zxx }
	
      Roland Stiner,
        NK2U
      ***@***.***
      
      
      
      
      
      
    
    On 3/10/2022 6:36 PM, tallman69 wrote:
    
    
      
      I have been running Open Shell v4.4.169-beta on the
        latest build of Windows 11 with no problems. However, before I
        installed Open Shell I installed ExporerPatcher and configured
        it to my liking. ExplorerPatcher automatically performs the
        tasks, plus many more, outlined in the ASKVG link posted by
        MEdson71 above. The two programs seem to work well together. You
        can find ExplorerPatcher here:
      https://github.com/valinet/ExplorerPatcher
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        Triage notifications on the go with GitHub Mobile for iOS or Android.
        
        You are receiving this because you commented.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2335424",
"url": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2335424",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by NK2U** [comment]
Michael,
    
    ---The date you installed Open-Shell on Windows 11?  23/5/22.
    
    ---Which version of Open-Shell that you have used?  4.4.170
    
    ---Have you had any update yet of Windows 11 and has any update of
    Windows 11 caused any problem with the installed Open-Shell on your
    Windows 11 computer? No, none.  Just had a security
      update, that's all.
    
    
      
      
      
      
      
      
      
      
		@page { margin: 0.79in }
		p { margin-bottom: 0.1in; line-height: 115% }
		a:link { so-language: zxx }
	
      Roland Stiner,
        NK2U
      ***@***.***
      
      
      
      
      
      
    
    On 6/18/2022 11:56 AM,
      MichaelIsGreat123 wrote:
    
    
      
      Check this video: "(May 28, 2022) Change the Windows
        11 start menu with Open-Shell - May 28, 2022 [by Sam Hayden] -
        YouTube" at https://www.youtube.com/watch?v=02RWTwZc9K0
        In this video, the guy uses Open-Shell 4.4.160 and he has no
        problems at all installing successfully Open-Shell on Windows
        11.
      I currently cannot at all test this scenario because
        I am still on Windows 10. However, if among you, there are
        people who recently successfully installed Open-Shell on Windows
        11, it would be great if you would not mind leaving a comment on
        this posting and indicate:
        ---The date you installed Open-Shell on Windows 11?
        ---Which version of Open-Shell that you have used?
        ---Have you had any update yet of Windows 11 and has any update
        of Windows 11 caused any problem with the installed Open-Shell
        on your Windows 11 computer?
        Thanks in advance for any feedback that you will share.
      Kind Regards
        Michael
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you commented.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2977175",
"url": "https://github.com/Open-Shell/Open-Shell-Menu/discussions/894#discussioncomment-2977175",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Bathsheba** [comment]
I installed the regular release on a fresh Windows 11 PC today and it seems to work fine.
I did only replace the start button and menu, not requesting changes to File Explorer.
Thanks devs! 

PS maybe drop support for Internet Explorer changes, now that that browser is itself no longer supported?

**Reply by blackcrack** [reply]
Hi @Bathsheba 
could you may make a Screenshot and put it on your Comment ?
because one picture say's more as thousand words .. you see :)
(what's me interested, the size of the Startbutton, if' it over the WindowsStartbutton button and so on...)
and also the version of the winNT, if you put in the cmd "winver" and put it by side of the Startbutton, then a Screenshot for see both :) (Don't forget to overpainting your registration name )
best :)

**Comment by ghost** [comment]
i can not wait until release version for 22H2

**Comment by cococha50** [comment]
I noticed that there is no glass blur .
That's too bad. Microsoft sure does like to hide access to the old code,
or perhaps they even deleted that aero glass code now?

Maybe Open-Shell coders can find a way to get it back.
I  tried "ExplorerPatcher" for the first time.
Unless I missed one of the settings,
I can't get Aero or fake glass to work with it.

**Reply by blackcrack** [reply]
well you can support our community also for find programers who be in able for spend an couples codeparts ;)
speak in the Forums about OpenShell and tell they are, the 0pen-Shell Community could have more programmers :)
